dunwoody_disaster/CharacterFactory.py

```diff
@@ -39,26 +45,26 @@ def set_health(self, health: int):
         if self.maxHealth == 0:
             percentage = 0
         else:
-            percentage = (health // self.maxHealth) * 100
-        self.health_lbl = QLabel(f"Health: {self.curHealth}")
+            percentage = (health / self.maxHealth) * 100
+        self.health_lbl.setText(f"Health: {self.curHealth}")
```

Comment:
Should we guard these functions from returning negative values? I'm assuming we don't want to have someone with a negative meter.

Reply:
Yuh